Actors/VisualObjects/VisualObjects.ActorService/Properties/AssemblyInfo.cs

```diff
@@ -1,33 +1,25 @@
-﻿// ------------------------------------------------------------

```

Comment:
Why was the license removed here? The resarper rules added it. Have you re-run resharper?